src/index.html

```diff
@@ -2,16 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     onsubmit="onSubmit(Submit)"
+     method="post">
```

Comment:
```suggestion
    <form
      class="form"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
onsubmit="onSubmit(Submit)" тут не потрібен, давай ще розставимо гарненько атрибути в тегах, згідно з чеклістом, ось приклад